lib/streamlit/elements/map.py

```diff
@@ -84,6 +84,8 @@ def map(
         size: None | str | float = None,
         zoom: int | None = None,
         use_container_width: bool = True,
+        width: int | None = None,
+        height: int | None = None,
```

Comment:
For `map.py` and `deck_gl_json_chart.py` it might make sense to also update the Python unit tests to at least ensure that the proto fields are set when values are passed for the arguments. Its covered by the e2e test but having (at least basic) unit tests help us to narrow down issues faster when we see test failures.